### DISCUSSION TITLE
How to change the html attribute lang?

### DISCUSSION THREAD

**Discussion by Hkr32** [facebook/docusaurus#5025] [Q&A] [answered]
By default it is equal to `lang="en"`
I need to replace with `lang="ru"`

**Comment by slorber** [comment, marked as answer]
You can use the i18n config even if you don't plan to translate your site:

https://docusaurus.io/docs/docusaurus.config.js#i18n

```js
module.exports = {
  i18n: {
    defaultLocale: 'ru',
  },
};
```

This will also automatically translate theme labels to Russian, using these translations (that you can override):
https://github.com/facebook/docusaurus/blob/master/packages/docusaurus-theme-classic/codeTranslations/ru.json
